src/bookmarks/background/addition.js

```diff
@@ -62,3 +63,14 @@ export async function createBookmarkByExtension(url) {
     index.addPageConcurrent({ pageDoc, bookmarkDocs: [bookmarkDoc] })
     db.put(bookmarkDoc)
 }
+
+export async function createBookmarkByTab(url, tabId) {
+    const storePageResult = await storePage({ tabId, url })
+    const { page } = await storePageResult.finalPagePromise
```

Comment:
No need to assign `page` here as it's never used; should show up as a lint warning.